src/main/java/com/autotune/experimentManager/utils/EMConstants.java

```diff
@@ -19,23 +19,25 @@
 import com.autotune.analyzer.utils.ServerContext;
 
 public class EMConstants {
-    private EMConstants() { }
+    private EMConstants() {
+    }
 
     public static class APIPaths {
-        private APIPaths() { }
         public static final String CREATE_EXPERIMENT_TRIAL = ServerContext.ROOT_CONTEXT + "createExperimentTrial";
         public static final String LIST_TRIAL_STATUS = ServerContext.ROOT_CONTEXT + "listTrialStatus";
+        public static final String EXPERIMENT_MANAGER_INITIATE_EXPERIMENT_TRAIL = ServerContext.ROOT_CONTEXT + "initiateExperimentTrial";
```

Comment:
What is the difference between `createExperimentTrial` and `InitiateExperimentTrial`?

Reply:
We cannot have both `createExperimentTrial` and `InitiateExperimentTrial` as it is very confusing for users. Please make sure that the functionality of both is folded into one API